docs/integrations/creating-nodes/build/reference/http-helpers.md

```diff
@@ -0,0 +1,98 @@
+# Making HTTP Requests
+
+!!! note "Programmatic style only"
+	The information in this document is for node building using the programmatic style. It doesn't apply to declarative style nodes.
+
+It's common to call external APIs or make HTTP requests to other services from a node.
+
+This plays a major role during node development, maintenance, and improvements.
+
+n8n provides a flexible helper for making HTTP requests that abstracts away most of the complexity.
+
+## How to use
+
+Call the helper inside the `execute` function:
+
+```typescript
+const response = await this.helpers.httpRequest(options);
+```
+
+Where `options` is an object in this format:
+
+```typescript
+{
+	url: string;
+	headers?: object;
+	method?: 'GET' | 'POST' | 'PUT' | 'DELETE' | 'HEAD';
+	body?: FormData | Array | string | number | object | Buffer | URLSearchParams;
+	qs?: object;
+	arrayFormat?: 'indices' | 'brackets' | 'repeat' | 'comma';
+	auth?: {
+		username: string,
+		password: string,
+	};
+	disableFollowRedirect?: boolean;
+	encoding?: 'arraybuffer' | 'blob' | 'document' | 'json' | 'text' | 'stream';
+	skipSslCertificateValidation?: boolean;
+	returnFullResponse?: boolean;
+	proxy?: {
+		host: string;
+		port: string | number;
+		auth?: {
+			username: string;
+			password: string;
+		},
+		protocol?: string;
+	};
+	timeout?: number;
+	json?: boolean;
+}	
+```
+
+`url` is required. The other fields are optional. The default method is `GET`.
+
+Some notes about the possible fields:
+
+- `body`: you can use a regular JavaScript object for JSON payload, a buffer for file uploads, an instance of FormData for `multipart/form-data`, and `URLSearchParams` for `application/x-www-form-urlencoded`.
+- `headers`: a key-value pair.  
+	* If `body` is an instance of `FormData` then n8n adds `content-type: multipart/form-data` automatically.  
+	* If `body` is an instance of `URLSearchParams`, then n8n adds `content-type: application/x-www-form-urlencoded`.  
+	* To override this behavior, set a `content-type` header.
+- `arrayFormat`: if your query string contains an array of data, such as `const qs = {IDs: [15,17]}`, the value of `arrayFormat` defines how n8n formats it.  
+	* `indices` (default):  `{ a: ['b', 'c'] }` as `a[0]=b&a[1]=c`  
+	* `brackets`: `{ a: ['b', 'c'] }` as `a[]=b&a[]=c`  
+	* `repeat`: `{ a: ['b', 'c'] }` as `a=b&a=c`  
+	* `comma`: `{ a: ['b', 'c'] }` as `a=b,c`
+- **auth**: Used for Basic auth. Provide `username` and `password`.
+- **disableFollowRedirect**: By default, n8n follows redirects. You can set this to false to prevent this from happening
```

Comment:
```suggestion
- **disableFollowRedirect**: By default, n8n follows redirects. You can set this to true to prevent this from happening.
```

"By default, n8n follows redirects"
So disableFollowRedirect is false by default, and then it should be set to true to prevent following redirects?